src/main/java/teammates/common/util/HttpRequestHelper.java

```diff
@@ -36,9 +36,9 @@ public static String getValueFromParamMap(Map<String, String[]> paramMap, String
     }
 
     /**
-     * Returns a HashMap object containing all the parameters key-value pairs from a URLFetchRequest object.
+     * Returns a Map object containing all the parameters key-value pairs from a URLFetchRequest object.
```

Comment:
`Returns a map <del>object</del> containing ...`
Map is an interface, technically, not object.